Src/coffeescript/cql-execution/src/example/browser/cql4browsers.js

```diff
@@ -3626,7 +3628,7 @@
       ref = this.element;
       for (i = 0, len = ref.length; i < len; i++) {
         el = ref[i];
-        obj[el.name] = el.execute(ctx);
+        obj[el.name] = el.exec(ctx);
```

Comment:
This was already changed from a previous commit (In the coffeescript file). However the cql4browsers wasn't regenerated. The name of the commit on cql4bonnie is "Removed execute from Instance class".